protocol/messenger_sync_raw_messages.go

```diff
@@ -188,6 +188,28 @@ func (m *Messenger) HandleSyncRawMessages(rawMessages []*protobuf.RawMessage) er
 				m.logger.Error("failed to handleSyncSavedAddress when HandleSyncRawMessages", zap.Error(err))
 				continue
 			}
+		case protobuf.ApplicationMetadataMessage_SYNC_ALL_KEYCARDS:
+			var message protobuf.SyncAllKeycards
+			err := proto.Unmarshal(rawMessage.GetPayload(), &message)
+			if err != nil {
+				return err
+			}
+			err = m.handleSyncKeycards(state, message)
+			if err != nil {
+				m.logger.Error("failed to handleSyncKeycards when HandleSyncRawMessages", zap.Error(err))
+				continue
+			}
+		case protobuf.ApplicationMetadataMessage_SYNC_KEYCARD_ACTION:
```

Comment:
if this type of message don't come from SyncDevices, then i think we don't need process it here, since method HandleSyncRawMessages only invoked during local pairing @saledjenic

Reply:
@qfrank you're perfectly right, thanks for the heads-up, updated that.